### PR TITLE
204 Messages for Tokens Durability

### LINKED ISSUE #204: [BONUS]: Alert User when access token only last one hour
### Description

Add a message on each services connexion if the tokens last only one hour

### To do

Improve users messages

### Usefulness

useful

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by 31Nathan**
# Updated feature
Inform the user of the durability of the access tokens.

### CODE REVIEW

**Review by STom6 — DISMISSED** [review]
lgtm

File: Server/app.js [hunk 1]
```diff
@@ -395,9 +395,9 @@ app.get('/twitch/auth/', function (req, res) {
       twhtokens.accessToken = req.session.passport.user.accessToken
       twhtokens.refreshToken = req.session.passport.user.refreshToken
       TwitchService.setUserData(twhtokens)
-      res.send("BACK TO THE APP NOW")
+      res.send("SUCCESS ! You can go back to the AREA Application.\n The access token will only last one hour.")
     } else {
-        res.send("l'authentification a échoué")
+        res.send("Authentification has failed. Pleas try again.")
```
Comment (JohanCDev) [inline Server/app.js]: PleasE

**Review by JohanCDev — CHANGES_REQUESTED** [review]

**Review by JohanCDev — APPROVED** [review]
lgtm

**Review by STom6 — APPROVED** [review]
lgtm

**Review by CedricCORGE — APPROVED** [review]
lgtm

**Pull request merged**